README.md

```diff
@@ -24,18 +24,21 @@ https://scalameta.org/metals/docs/contributors/getting-started.html
 The current maintainers (people who can merge pull requests) are:
 
 - Adrien Piquerez - [`@adpi2`](https://github.com/adpi2)
-- Alexey Alekhin - [`@laughedelic`](https://github.com/laughedelic)
 - Chris Kipp - [`@ckipp01`](https://github.com/ckipp01)
 - Gabriele Petronella - [`@gabro`](https://github.com/gabro)
+- Kamil Podsiadło - [`@kpodsiad`](https://github.com/kpodsiad)
+- Ólafur Páll Geirsson - [`@olafurpg`](https://github.com/olafurpg)
```

Comment:
Feel free to move me to the inactive list

Reply:
But I’m ok with staying on the active list as well. You maybe won’t get rid of me so easily 😅

Reply:
I am only removing the people that haven't participated in any discussion in the last year. I think I will leave you here and put on the pressure of maybe popping by with some amazing feature :smile: